### PR TITLE
Add gatewayapi-notifier recipe

### PR DESCRIPTION
**Pull request by PGLongo**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | [symfony/symfony-docs/pull/14463](https://github.com/symfony/symfony-docs/pull/14463)

Add recipe to use with GatewayAPI notifier [symfony/symfony/pull/38685](https://github.com/symfony/symfony/pull/38685)

### CODE REVIEW

File: symfony/gatewayapi-notifier/5.2/manifest.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "#1": "GATEWAYAPI_DSN=gatewayapi://AUTH_TOKEN@default?from=SENDER"
```
Comment (OskarStark) [inline symfony/gatewayapi-notifier/5.2/manifest.json:3]:
```suggestion
        "#1": "GATEWAYAPI_DSN=gatewayapi://TOKEN@default?from=SENDER"
```

**Review by OskarStark — CHANGES_REQUESTED** [review]

**Comment by OskarStark** [comment]
This should be moved to folder `5.3`

**Comment by PGLongo** [comment]
@OskarStark I've moved the recipe.

**Review by OskarStark — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
@symfony-flex-server review again please

**Comment by fabpot** [comment]
@symfony-flex-server review again please

**Comment by fabpot** [comment]
@PGLongo Can you rebase this pull request to get rid of the merge commit? Thank you.

**Comment by PGLongo** [comment]
@fabpot rebase done

File: symfony/gatewayapi-notifier/5.3/manifest.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "env": {
+        "#1": "GATEWAYAPI_DSN=gatewayapi://TOKEN@default?from=SENDER"
```
Comment (OskarStark) [inline symfony/gatewayapi-notifier/5.3/manifest.json:3]:
```suggestion
        "#1": "GATEWAYAPI_DSN=gatewayapi://TOKEN@default?from=FROM"
```

To stay consistent with the README

**Review by fabpot — APPROVED** [review]

**Pull request merged**